tests/test_scope.py

```diff
@@ -25,11 +25,6 @@ def test_basic():
     _assert_right_scopes()
 
 
-def test_correct_span():
```

Comment:
this test is no longer accurate since the `get_current_scope` inside a test has nothing to do with the span that was created by the pytest hook.

Reply:
yeah i think this test didn't make sense to begin with